src/NuGet.Core/NuGet.ProjectModel/Utf8JsonStreamReader.cs

```diff
@@ -220,6 +221,25 @@ internal bool ReadNextTokenAsBoolOrFalse()
             return false;
         }
 
+        internal bool ReadNextTokenAsBoolOrTrue(string propertyName)
```

Comment:
I now see that write if false wouldn't be a bug. 
Are we trying to save on package spec size here?

Reply:
that was my suggestion, yes. Most projects now (i'm guessing, I didn't look at telemetry) are SDK style, so we can avoid CPU cycles parsing an additional property and its value when the majority of the time it's going to have that value.

Reply:
it's not a huge perf impact and it's the type of code that can easily lead to bugs. 

It's ok if I'm the only person with the concern though.